sfge/src/main/java/com/salesforce/rules/PathBasedRuleRunner.java

```diff
@@ -142,7 +144,10 @@ private void executeRulesOnPaths(List<ApexPath> paths) {
                         // to the list of such
                         // objects.
                         if (ruleThrowable instanceof FlsViolationInfo) {
-                            incompleteThrowables.add((FlsViolationInfo) ruleThrowable);
+                            flsViolationInfos.add((FlsViolationInfo) ruleThrowable);
+                        } else if (ruleThrowable instanceof MultipleMassSchemaLookupInfo) {
+                            // FIXME: PR incoming with refactors to this portion
+                            mmsLookupInfos.add((MultipleMassSchemaLookupInfo) ruleThrowable);
```

Comment:
There's some duplication here that can be streamlined.

Reply:
I'm assuming the subsequent PR will involve refactoring this so it doesn't require a Set of MMS violations specifically?

Reply:
Right, I'm trying to move the ownership of conversion to the ViolationInfo object so that this portion can work on the higher-level interface.